src/compile/axis/parse.ts

```diff
@@ -75,18 +75,26 @@ function parseAxis(channel: Channel, model: Model, isGridAxis: boolean): VgAxis
   };
 
   // 1.2. Add properties
-  [
+  const axisProperties:(keyof Axis)[] = [
```

Comment:
Move this to src/axis.ts and name it like a proper constant (AXIS_PROPERTIES). 

Also, read [style guide](https://basarat.gitbooks.io/typescript/docs/styleguide/styleguide.html) if you haven't.